arch/arm/soc/nordic_nrf/nrf52/power.c

```diff
@@ -4,8 +4,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#define LOG_MODULE_NAME soc_power
```

Comment:
What about nrf_power? Such name will suggest where look for the code which sent the message.

Reply:
or nrf_soc_power